src/annotator/components/toolbar.js

```diff
@@ -121,21 +120,21 @@ export default function Toolbar({
         />
       )}
       {!useMinimalControls && (
-        <ToolbarButton
-          label="Show highlights"
-          icon={showHighlights ? 'show' : 'hide'}
-          selected={showHighlights}
-          onClick={toggleHighlights}
-        />
-      )}
-      {!useMinimalControls && (
-        <ToolbarButton
-          label={
-            newAnnotationType === 'note' ? 'New page note' : 'New annotation'
-          }
-          icon={newAnnotationType === 'note' ? 'note' : 'annotate'}
-          onClick={createAnnotation}
-        />
+        <div className="annotator-toolbar-buttonbar">
```

Comment:
This change is to add a parent/wrapper element that can handle the spacing of the buttons (margins).